package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@looker/embed-sdk",
-  "version": "1.6.0",
+  "version": "1.6.1",
```

Comment:
Hmm. I guess a patch release is OK. I think extending the API is typically a minor version but this is all types so 🤷